src/types.ts

```diff
@@ -40,3 +45,44 @@ type AdditionalErrors = {
 }
 
 export type FormErrors = RequiredErrors & AdditionalErrors
+
+export type ParseFn<FieldValue, InputValue = unknown> = (
```

Comment:
Can't we reuse final-form types ?

Reply:
There are no types for this in final-form and react-final-form

Reply:
I think we can extract them 

```ts
  parse?: FieldProps<FieldValue, InputValue>['parse'],
  format?: FieldProps<FieldValue, InputValue>['format']
```